app/src/components/map/MapFeatureList.tsx

```diff
@@ -48,35 +58,81 @@ const MapFeatureList: React.FC<MapFeatureListProps> = (props) => {
     activeFeatureState[1](null);
   };
 
+  const deleteListItem = (index: number) => {
+    const location = get(values, 'location');
+    const oldFeatureList = get(values, 'location.geometry');
+    const newFeatureList = recalculateFeatureIds(
+      oldFeatureList.filter((f: Feature, i: number) => i !== index)
+    );
+
+    location.geometry = newFeatureList;
+    setFieldValue('location', location);
+
+    // Reset the hover state
+    mouseLeaveListItem();
+
+    // To keep the map happy, we need to update the maskState
+    maskState[1](() => {
+      const oldState = maskState[0];
+      const newState = oldState.filter((f: boolean, i: number) => i !== index);
+      return newState;
+    });
+  };
+
+  interface FeatureItemProps {
+    properties?: any;
+    feature: Feature;
```

Comment:
Question: what type of Feature are we tracking here? 

I know of several types "Feature" from different deps. (Turf, geojson, maplibre)
I have found it pretty confusing trying to figure out in which state the current feature is.

Reply:
Currently we only accept Polygons and MultiPolygons. However, as you mention... they are both features as well. So you can think of a "Feature" as JSON schema... and the Geometry is where you get the different kinds of shapes (Polygons etc...).

As far as I know, Turf and maplibre both follow the geojson standard. https://geojson.org/